modules/imaging_uploader/php/NDB_Menu_Filter_imaging_uploader.class.inc

```diff
@@ -596,14 +596,12 @@ class NDB_Menu_Filter_Imaging_Uploader extends NDB_Menu_Filter_Form
      */
     function _getFileList($data)
     {
-
         $files = array();
-
         foreach ($data as $row) {
+            $uploadStatus = $row[1];
             $filePath     = $row[5];
+            $mincInserted = $row[10];
```

Comment:
@davidblader is there a risk of these hard coded values being different from a project to another ? if so, a redmine ticket should be issued in parallell to flag this for change in a minor version